web_src/less/themes/theme-arc-green.less

```diff
@@ -1107,7 +1107,10 @@ input {
 .lines-code ol,
 .lines-num .hljs,
 .lines-code .hljs {
-    background-color: #2a2e3a !important;
+    background-color: #2a2e3a;
```

Comment:
This change seems unrelated. This CSS doesn't apply to diffs as far as I know (at least I didn't see it in testing).